### DISCUSSION TITLE
Add an alpha argument to setAll()

### DISCUSSION THREAD

**Discussion by sidewayss** [color-js/color.js#468] [Ideas]
`setAll()` does not currently set alpha.  It would be a tiny change to the code, an optional additional argument, and (barring a need for validation against invalid/undefined values) one additional line of code:
`color.alpha = alpha;`

**Comment by LeaVerou** [comment]
I think that makes sense. `setAll()` should conceptually be able to set everything that `set()` can, and `set()` *can* set alpha. What do others think?
